avalanche/logging/wandb_logger.py

```diff
@@ -28,17 +27,19 @@ class WandBLogger(StrategyLogger):
     logged to a dedicated Weights & Biases project dashboard.
     """
 
-    def __init__(self, init_kwargs: dict, interactive: bool = False):
+    def __init__(self, project_name: str, run_name: str, params: dict):
```

Comment:
Can you make `params` optional by setting it by default to `{}`?

Reply:
Sure, yes